rfp-proposals/RFP-proposal：“Interstellar album”.md

```diff
@@ -0,0 +1,97 @@
+
+# Open Grant Proposal: `Project Title`
+
+**Name of Project:**Interstellar album
+
+**Proposal Category:**  `app-dev`
+
+**Proposer:** `Jonney hu`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:**  "Yes" 
+
+# Project Description
+
+- Interstellar album is a cloud photo album application. The starting point is to help users solve the problem of insufficient space when storing photos and videos, as well as high storage costs.Interstellar album can also prevent photos from being lost in mobile phones. Even if the phone is lost, use other devices to log in to Interstellar album, upload it to Interstellar album and retrieve it. All photos and videos are distributed and decentralized. All data can be used only by the users themselves, there will be no security incidents such as leaks, and the security and privacy of user data will be increased.
+- Interstellar album is based on blockchain, based on the application developed by Filecoin, perfectly integrated into the blockchain and Filecoin features.
+
+
+## Value
+
+- If Interstellar album is developed and popularized, users will learn about IPFS through Interstellar album and learn about Filecoin. Then it will build more nodes for IPFS and filecoin, which will break the current problem facing the cloud photo album market: security, privacy. . Become one of the pioneering applications of IPFS and Filecoin in the market, and increase the popularity of IPFS and Filecoin.
+- Because Interstellar album is built on top of Filecoin's slice storage technology, we need Filecoin API support.
```

Comment:
Yes, because of how unstable the lotus and go-filecoin APIs are, I think it is a little too early for this project at this time.